ml/dgl_utils.py

```diff
@@ -0,0 +1,253 @@
+# classes for converting a dataframe or Graphistry Plottable into a DGL
+import pandas as pd
+import dgl
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import umap
+import numpy as np
```

Comment:
just a style thing, try to do one-line & sorted imports

Reply:
100%